README.md

```
@@ -0,0 +1,140 @@
+# 🎄 EveryVent
+
+<div align="center">
+
+### 매일의 이벤트를 받아보세요, **EveryVent!**
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)